project/plugins.sbt

```diff
@@ -15,21 +15,22 @@
  */
 
 addSbtPlugin("com.github.gseitz"     % "sbt-release"     % "1.0.6")
-addSbtPlugin("org.scalariform"       % "sbt-scalariform" % "1.7.1")
-addSbtPlugin("de.heikoseeberger"     % "sbt-header"      % "2.0.0")
+addSbtPlugin("org.scalariform"       % "sbt-scalariform" % "1.8.1")
+addSbtPlugin("de.heikoseeberger"     % "sbt-header"      % "4.0.0")
 addSbtPlugin("org.foundweekends"     % "sbt-bintray"     % "0.5.1")
+addSbtPlugin("org.xerial.sbt"        % "sbt-sonatype"    % "2.0")
 addSbtPlugin("com.jsuereth"          % "sbt-pgp"         % "1.1.0-M1")
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox"     % "0.3.0")
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox"     % "0.3.1")
```

Comment:
we just released 0.3.2 - perhaps we should add updating this one to the release script :)